activerecord/test/cases/connection_adapters/schema_cache_test.rb

```diff
@@ -39,6 +41,10 @@ def test_yaml_loads_5_1_dump
           assert cache.data_sources("posts")
           assert_equal "id", cache.primary_keys("posts")
         end
+
+        assert_queries 1 do
+          assert_equal 1, cache.indexes("posts")
+        end
```

Comment:
Not sure what to do for old schema caches. They'll have to execute a query at some point.

Reply:
I'm also imagining we'll need serialization tests to ensure a 6.0 dump can be read on 5.2 and a 5.2 dump can be read on 6.0, right?